src/scss/components/form/_index.scss

```diff
@@ -1 +1,157 @@
+@import "config";
+
+// for Firefox
 @import "reset"; // ./_reset.scss
+
+
+// ---------- CONFIGURATION ----------
+$form-asterisk-size: 8px; // OK by design, 2019-04-09
+$form-asterisk-gap: 4px;
+$form-asterisk-offset: $form-asterisk-size + $form-asterisk-gap;
+
+// ---------- RESETS ----------
+.beta-hxForm {
+  fieldset {
+    border: none;
+    padding: 0;
+  }
+
+  input {
+    @include hxInput(reset);
+  }
+
+  // [011] .beta-hxForm label
+  //label {
+  //  color: inherit;
+  //  font-size: inherit;
+  //  font-weight: inherit;
+  //  padding: 0;
+  //}
+
+  // TODO: Is a <legend> always a field name?
+  legend {
+    border: none;
+    color: inherit;
+    display: block;
+    font-size: 1rem;
+    font-weight: 400;
+    line-height: 1;
+    padding: 0;
+    width: 100%;
+  }
+
+  select {
+    // TODO: Is this still needed? #Moz.reset.invalid();
+    box-shadow: none;
+    border: 0;
+    outline: 0;
+  }
+
+  textarea {
+    font: inherit;
+  }
+}
+//end:RESETS
+
+// ---------- BASE ----------
+*:disabled {
+  cursor: not-allowed;
+}
+//end:BASE
+
+// ---------- FORM COMPONENTS ----------
+.hxPrefix, // $[100]
+.hxInfix, // [12]:[45]
+.hxSuffix { // [65]lbs
+  font-size: 1rem;
+  color: $gray-900;
+}
+
+.beta-hxForm {
+  > * {
+    margin: 1rem 0; // FIXME: solve with spacing system
+
+    &:first-child {
+      margin-top: 0; // FIXME: solve with spacing system
+    }
+
+    &:last-child {
+      margin-bottom: 0; // FIXME: solve with spacing system
+    }
+  }
+
+  fieldset {
+    > * {
+      margin: 0.25rem 0; // FIXME: solve with spacing system
+
+      &:first-child {
+        margin-top: 0; // FIXME: solve with spacing system
+      }
+
+      &:last-child {
+        margin-bottom: 0; // FIXME: solve with spacing system
+      }
+    }
+
+    > legend {
+      color: $gray-800;
+      font-size: 0.875rem; // ~14px
+    }
+
+    > label {
+      display: block;
+      font-size: 1rem; // NOTE: this may change with checkboxes and radios
+    }
+  }
+
+  select,
+  input[type="range"] {
+    display: block;
+    width: 100%;
+  }
+}
+
+.beta-hxField {
+  // placeholder for potential styles
+}
+
+.beta-hxFieldName {
+  @include hxFieldName;
+}
+//end:COMPONENT
+
+/* ---------- FORM HELPERS ---------- */
+
+/*
+  Screen readers already announce if a field is required, but we need
+  to display a required asterisk to sighted users in a way that avoids
+  unnecessary text announcements by a screen reader.
+
+  If we apply `content: '*'` to `::before`, a screen reader would
+  announce "star" as if it were part of the coded label text.
+
+  Obviously, this is underiable.
+
+  By inlining an SVG of the asterisk into `background-image`, we can
+  provide sighted users an indicator while allowing screen readers to
+  correctly announce text content (as they already do).
+*/
```

Comment:
For bulk comment please follow "///" to the complete section

Reply:
Ok comment updated.